### DISCUSSION TITLE
⭐️ Starstruck Badge Exchange ⭐️

### DISCUSSION THREAD

**Discussion by Rantoryu** [Schweinepriester/github-profile-achievements#778] [Q&A] [answered]
Help me get starstruck badge, I will ⭐️**star**⭐️ your repo back - provide link in the comment please!
https://github.com/Rantoryu/Personal_Website 

_**Note:** You can see who's actually starred your repo (e.g. "returned the favor") by appending /stargazers to the end of your repo's URL. For example: https://github.com/Rantoryu/Personal_Website/stargazers_

**Comment by Schweinepriester** [comment, marked as answer]
See https://github.com/Schweinepriester/github-profile-achievements/discussions/722

**Comment by pulkitxm** [comment]
https://github.com/Pulkitxm/Delta-Apna-College
